### ISSUE TITLE
AI casts Path to Exile and targets mana dork in early game

### ISSUE THREAD

**Issue by MorihiroS** [Card-Forge/forge#5907]
**Describe the bug**
AI casts Path to Exile and targets Noble Hierarch at Turn 1. It only helps the opponent.

**To Reproduce**
1. Player plays Forest and casts Noble Hierarch
2. AI plays Plains, then it casts Path to Exile and targets Noble Hierarch
3. Player finds free basic land

**Expected behavior**
AI shouldn't use Path to Exile versus mana dorks

**Desktop**
OS: Windows 10
Java Version: 1.8.0_202-b08
Forge Version: 1.6.64-SNAPSHOT-08.10

**Screenshots**

![path](https://github.com/user-attachments/assets/542a63f3-45e0-4361-b2af-ff13b5293947)

**Comment by MorihiroS** [comment]
@Hanmac, need AI label, keep label and assign someone, please

**Comment by Hanmac** [comment]
imo Mana Dorks might be way too dangerous, so I don't think it is that of a bad move

**Comment by MorihiroS** [comment]
@Hanmac, I strongly disagree. First off, it exchanges dork for land, which doesn't actually slow down the player. Second, it thins the deck, giving the player an advantage. Third, it allows the player to find a basic land of an additional color and stabilize his colored mana base.
I even asked other people who play the format modern, and my opinion was confirmed. Path to Exile is designed for big threats or when the threat shouldn't hit the graveyard.

**Comment by 8BitAvenger** [comment]
> @Hanmac, I strongly disagree. First off, it exchanges dork for land, which doesn't actually slow down the player. Second, it thins the deck, giving the player an advantage. Third, it allows the player to find a basic land of an additional color and stabilize his colored mana base. I even asked other people who play the format modern, and my opinion was confirmed. Path to Exile is designed for big threats or when the threat shouldn't hit the graveyard.

Hard agree here. Considering the player receives a land on the battlefield in exchange for their mana dork, it's really hard to argue this is anything but horrendous strategy. It's not a 1 mana kill spell, it's not a bolt or shock, it's effectively do nothing except help your opponent for the opportunity cost of not being able to remove an threat that's worth trading for mana later. This would be an actual discussion with Swords to Plowshares, but it's just bad with Path to exile.

**Comment by MorihiroS** [comment]
Same problem with Erode and Assassin's Trophy

<img width="1361" height="1013" alt="Image" src="https://github.com/user-attachments/assets/10a2062c-fd64-4b40-831b-89e78afad943" />
<img width="1331" height="705" alt="Image" src="https://github.com/user-attachments/assets/e11e816c-c814-4e9d-a797-215b6f284467" />
<img width="1917" height="1006" alt="Image" src="https://github.com/user-attachments/assets/330e3743-0103-4fd2-ae9e-89314d12418d" />
